packages/@apphosting/common/src/index.ts

```diff
@@ -0,0 +1,25 @@
+// Options to configure the build of a framework application
+export interface BuildOptions {
+  // command to run build script (e.g. "npm", "nx", etc.)
```

Comment:
This may be too early to discuss but does this support go frameworks?

Reply:
At first glance I don't why not - the BuildOptions interface is essentially just allowing something (whether that's the monorepo buildpack or a go buildpack) to configure how the build is run.